modules/osdk-golang-run-in-cluster.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-golang-tutorial.adoc
+
+[id="osdk-golang-run-in-cluster_{context}"]
+= Running as a deployment inside the cluster
+
+You can run your Operator project as a deployment inside of your cluster.
+
```

Comment:
```suggestion

.Before Beginning

Certain image references throughout the default project should be updated to supported image tags:
- In the top-level `Dockerfle`, change the "builder" and "runner" images from `golang:1.15` and `gcr.io/distroless/static:nonroot` to `registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7` and `registry.access.redhat.com/ubi8/ubi-minimal:latest`, respectively. Additionally, remove the `USER: nonroot:nonroot` directive as it is not required by the recommended supported image.
- In `config/default/manager_auth_proxy_patch.yaml`, change the `gcr.io/kubebuilder/kube-rbac-proxy` image reference to `quay.io/openshift/origin-kube-rbac-proxy:{product-version}`.

```

Reply:
Made them just the first step in the procedure.

Reply:
And is `registry.redhat.io/openshift4/ose-kube-rbac-proxy` the correct downstream image, instead of the quay.io OKD one?